build-profiling-ffi.sh

```diff
@@ -4,6 +4,11 @@
 # under the Apache License Version 2.0. This product includes software developed
 # at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
 
+# Location to place all artifacts
+if [ -z $CARGO_TARGET_DIR ] ; then
+    export CARGO_TARGET_DIR=$PWD/target
```

Comment:
This changes where the `target` directory is, as you can run `cargo build` from a subdirectory and it would create the `target` dir from now on in this subdirectory. Before it would have been created relative to the `Cargo.toml`, now it would be relative to the `$PWD`. 
OTOH: The `build-profiling-ffi.sh` is only working when called from same directory where it is. If you go to the `profiling-ffi` directory and call it via `../build-profiling-ffi.sh` it fails anyway, so this might not be a problem after all.

Reply:
It's a good find, we should support it at some point. Since it doesn't work today and this doesn't change it, it doesn't block this PR.